test/api/invite.test.js

```diff
@@ -47,9 +47,12 @@ test('tribes.invite', async t => {
       text: 'Welcome new person!',
       recps: [groupId]
     }
-    const { key: greetingKey } = await p(kaitiaki.publish)(greetingContent)
-    await p(replicate)({ from: kaitiaki, to: newPerson, live: false, name })
-    const greetingMsg = await p(Getter(newPerson))(greetingKey)
+    const { key: greetingKey } = await p(kaitiaki.publish)(greetingContent).catch(t.fail)
+    await p(replicate)({ from: kaitiaki, to: newPerson, live: false, name }).catch(t.fail)
+    const greetingMsg = await p(Getter(newPerson))(greetingKey).catch(err => {
+      console.error('error on getting greeting msg:', err)
```

Comment:
Failing with:

```
% node test/api/invite.test.js
TAP version 13
# tribes.invite
ok 1 creates group
ok 2 kaitiaki sent invite
kaitiaki  ─> new person
  1 [?]
  2 [?]
  3 group/add-member  ─ new person, undefined
  4 [?]
db.rebuild (ﾉ´ヮ´)ﾉ*:･ﾟ✧
db.rebuild finished @HNyZvR8yt99F1SQYqWYxPv/bWhPwE1RgFFG/rTaiD7Y=.ed25519
db.rebuild (ﾉ´ヮ´)ﾉ*:･ﾟ✧
db.rebuild finished @NjoSnRNQuqicFy+RwG58op9at/rbAcWh4Hkuf5WN5QM=.ed25519
error on getting greeting msg: Error: flumeview-level.get: index for: %bAKnWZZZXjpEQgWahTf9BsuCsqGDfW4TlDUtk+TYj5M=.sha256pointed at:3517but log error
    at /home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/flumeview-level@4.0.4/node_modules/flumeview-level/index.js:170:17
    at /home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/ssb-db@20.4.1/node_modules/ssb-db/minimal.js:54:7
    at /home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/flumelog-offset@3.4.4/node_modules/flumelog-offset/frame/recoverable.js:48:11
  Error: failed to get decrypted msg: %bAKnWZZZXjpEQgWahTf9BsuCsqGDfW4TlDUtk+TYj5M=.sha256
    at /home/me/prj/ssb/ssb-tribes/test/api/invite.test.js:85:35
    at /home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/ssb-db@20.4.1/node_modules/ssb-db/create.js:84:19
    at /home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/flumeview-level@4.0.4/node_modules/flumeview-level/index.js:180:15
    at /home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/ssb-db@20.4.1/node_modules/ssb-db/minimal.js:52:7
    at /home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/ssb-db@20.4.1/node_modules/ssb-db/minimal.js:41:35
    at AsyncJobQueue.runAll (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/ssb-db@20.4.1/node_modules/ssb-db/util.js:197:32)
    at Array.<anonymous> (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/ssb-db@20.4.1/node_modules/ssb-db/minimal.js:41:22)
    at chainMaps (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/ssb-db@20.4.1/node_modules/ssb-db/minimal.js:61:14)
    at /home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/flumedb@2.1.8/node_modules/flumedb/index.js:119:14
    at /home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/flumelog-offset@3.4.4/node_modules/flumelog-offset/inject.js:93:9
not ok 3 Error: flumeview-level.get: index for: %bAKnWZZZXjpEQgWahTf9BsuCsqGDfW4TlDUtk+TYj5M=.sha256pointed at:3517but log error
  ---
    operator: fail
    at: <anonymous> (/home/me/prj/ssb/ssb-tribes/test/api/invite.test.js:54:9)
    stack: |-
      Error: Error: flumeview-level.get: index for: %bAKnWZZZXjpEQgWahTf9BsuCsqGDfW4TlDUtk+TYj5M=.sha256pointed at:3517but log error
          at Test.assert [as _assert] (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:273:48)
          at Test.bound [as _assert] (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:85:17)
          at Test.fail (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:366:7)
          at Test.bound [as fail] (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:85:17)
          at /home/me/prj/ssb/ssb-tribes/test/api/invite.test.js:54:9
          at async Test.<anonymous> (/home/me/prj/ssb/ssb-tribes/test/api/invite.test.js:52:25)
  ...
not ok 4 TypeError: Cannot read properties of undefined (reading 'value')
  ---
    operator: fail
    at: Test.<anonymous> (/home/me/prj/ssb/ssb-tribes/test/api/invite.test.js:69:7)
    stack: |-
      Error: TypeError: Cannot read properties of undefined (reading 'value')
          at Test.assert [as _assert] (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:273:48)
          at Test.bound [as _assert] (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:85:17)
          at Test.fail (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:366:7)
          at Test.bound [as fail] (/home/me/prj/ssb/ssb-tribes/node_modules/.pnpm/tape@4.16.2/node_modules/tape/lib/test.js:85:17)
          at Test.<anonymous> (/home/me/prj/ssb/ssb-tribes/test/api/invite.test.js:69:7)
  ...

1..4
# tests 4
# pass  2
# fail  2
```

Reply:
This error was correct to happen, but the way I wrote the tests the real culprit was not showing their face. SPOILER - it's because you were not decrypting the messages... because you messed with the keystore.

I've pushed changes so this in the error you get now:

![image](https://github.com/ssbc/ssb-tribes/assets/2665886/65f5cbd3-88e8-41bf-abf3-766e0a6a4415)

Reply:
as mentioned below, fixed in https://github.com/ssbc/ssb-tribes/commit/764200f5daca62f1839c73df4189158dce36851a